draft-ietf-dots-requirements.md

```diff
@@ -277,14 +277,14 @@ another vector for the very attacks it's meant to help fight off. DOTS clients
 must be able to authenticate DOTS servers, and vice versa, to avoid exposing new
 attack surfaces when deploying DOTS; specifically, to prevent DDoS mitigation in
 response to DOTS signaling from becoming a new form of attack. In order to
-provide this level of proteection, DOTS agents must have a way to negotiate and
+provide this level of protection, DOTS agents must have a way to negotiate and
```

Comment:
I just merged this change in from Flemming's pull request (PR #65), you can drop it from the next revision of this PR.

Reply:
Works for me